pkg/cluster/cluster.go

```diff
@@ -243,6 +257,10 @@ type ClusterSpec struct {
 	ExistingConfig *ExistingConfig `json:"existingConfig,omitempty"`
 	// Standby setting when role is standby
 	StandbySettings *StandbySettings `json:"standbySettings,omitempty"`
+	// Set default hba auth for replication user and superuser.
```

Comment:
`Define the mode of the default hba rules needed for replication by standby keepers (the su and repl auth methods will be the one provided in the keeper command line options)`

Reply:
Done